index.js

```diff
@@ -0,0 +1,209 @@
+const typeScriptSettings = {
```

Comment:
I think we should disable `consistent-return` ([info](https://eslint.org/docs/rules/consistent-return)) when using TypeScript. It doesn't add any value. TypeScript will already infer/check all returns and see if they make sense.

Now you're forced to do this:
```js
function foo() {
  if (condition) {
    return 'value';
  }
  return undefined; // this doesn't add anything useful
}
```